crates/bevy_core/src/time/time.rs

```diff
@@ -33,7 +79,12 @@ impl Time {
         self.update_with_instant(Instant::now());
     }
 
-    pub(crate) fn update_with_instant(&mut self, instant: Instant) {
+    /// Update time with a specified [`Instant`]
+    ///
+    /// This method is for use in tests, and should generally not be called on the [`Time`] resource
+    /// as part of your app. Doing so will conflict with the `time_system` called by
+    /// [`CorePlugin`](crate::CorePlugin), and result in inaccurate performance measurements.
```

Comment:
```suggestion
    /// This method is provided for use in tests. Calling this method on the [`Time`] resource in a normal application
    /// will most likely result in inaccurate timekeeping, as the resource is ordinarily managed by the [`CorePlugin`](crate::CorePlugin).
```

Reply:
Thank you for the suggestion! I updated the documentation accordingly.

Reply:
Can you add the second sentence as a disclaimer to `Time::update` as well?

Reply:
Good idea, agreed.

Reply:
Sure. Done!